server/app/actors/player-actor.coffee

```diff
@@ -1,10 +1,14 @@
 debug = require('debug')('sh:player-actor')
+_ = require('underscore')
+
+PLAYER_COLORS = ["red", "blue", "green", "yellow", "white"]
 
 class PlayerActor
   constructor: (@manager, @id) ->
     @type = 'player'
     @x = 1
     @y = 1
+    @color = _.sample(PLAYER_COLORS)
```

Comment:
I think round robin would be better algorithm for assigning colors. Current has 20% change that the first two players have same color.